WebPortal/Pages/Catalog/DiscoverySearchLanding.razor

```diff
@@ -127,8 +127,15 @@
 	{
 		foreach (var browserFile in e.GetMultipleFiles())
 		{
-			var stream = browserFile.OpenReadStream();
-			await CatalogImportService.Import(stream);
+			try
+			{
+				var stream = browserFile.OpenReadStream(int.MaxValue);
+				await CatalogImportService.Import(stream);
+			}
+			catch (Exception ex)
+			{
+				Console.WriteLine(ex.Message);
```

Comment:
I don't think the console outputs anywhere useful in Azure, so this should probably be logged with a proper logger.

Reply:
Agree - I would move this to logging

Reply:
Done